tests/unit/LimitedRestorationPanel.spec.ts

```diff
@@ -15,12 +15,14 @@ localVue.use(VueRouter)
  */
 function createDefaultComponent (
   currentDate: string = '2023-02-03',
-  expiryDate: string = null
+  expiryDate: string = null,
+  maxNumberOfMonths: number = 24
```

Comment:
Maybe you shouldn't default this, so you can test the component's default. Does this work if you default to null here? Then you can add a test to verify a different max.

Reply:
It doesn't work! I believe the default value of 24 is not being set properly. If I set it manually like this, it works just fine. Shall I leave it as is or try to work around it?

Reply:
I think the problem is "null" is being passed in, which is different than the prop not being passed at all.

Leave it.